### DISCUSSION TITLE
Plotting on two PC's

### DISCUSSION THREAD

**Discussion by cjcoopertag** [Chia-Network/chia-blockchain#4186] [Q&A]
Hello everyone, so I am plotting on two different PC's with one being my node.  I have modified the config file so there will be no conflicts.  My question is how do I turn off the farming on the second pc and only have it plotting?  Also, currently my node does not see the plots that have been completed on my second pc therefore it's as they are working independantly even though they are connected via wallet.  

Would it be best that once my drive is filled on my 2nd pc to then pull that drive and attach it to my main node for farming?  

My node setup is a Ryzen 5900X, 64GB Ram and I am running 12plots in parallel.

**Comment by amirdt22** [comment]
Check out https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines and https://github.com/Chia-Network/chia-blockchain/wiki/Good-Security-Practices-on-Many-Machines

Briefly, plotting does not require the farmer/node running.  Harvesting doesn't either but you may want to harvest on your second plotting machine instead of filling a drive and moving it to the main machine (although that would work).  The first link details how to set up the remote harvester.

**Comment by cjcoopertag** [comment]
Thank you.  I will work on this.  I do want to confirm that it will farm the plots that have been completed on my main system?  I currently have plots on two machines and they are not syncing, so if I complete the instructions that you sent to me should I assume that will resolve that issue?  As of now I have 44 plots on one machine "node" and 32 on the other.  The second machine is plotting and harvesting just as the first one but I do have the 2nd machine not set up as a node.  I just want to be certain that my main system "node" recognizes all plots for farming regardless of which machine they are stored on.  
Sincerely, 
Chris    

    On Saturday, May 8, 2021, 09:19:24 AM CDT, Amir Tahvildaran ***@***.***> wrote:  
 
 


Check out https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines and https://github.com/Chia-Network/chia-blockchain/wiki/Good-Security-Practices-on-Many-Machines

Briefly, plotting does not require the farmer/node running. Harvesting doesn't either but you may want to harvest on your second plotting machine instead of filling a drive and moving it to the main machine (although that would work). The first link details how to set up the remote harvester.

**Reply by amirdt22** [reply]
Check out the mention of logs in the farming on many machines article.  This is currently how you will verify they are "syncing", meaning the main node is getting info from the harvester, and the harvester is checking X plots.  "syncing" does not currently mean the farm summary information will show all of the plots you have on all of the remote harvesters. 

Also see: https://github.com/Chia-Network/chia-blockchain/issues/3875

It sounds like there is a patch coming soon that will relay this information.

**Comment by cjcoopertag** [comment]
Thank you, I look forward to it.  We are actually revamping our system and building a server where all drives will be housed.  We will then have two setups plotting to the server and one farming from the server.  
Sincerely, 
Chris   

    On Saturday, May 8, 2021, 07:40:16 PM CDT, Amir Tahvildaran ***@***.***> wrote:  
 
 


Check out the mention of logs in the farming on many machines article. This is currently how you will verify they are "syncing", meaning the main node is getting info from the harvester, and the harvester is checking X plots. "syncing" does not currently mean the farm summary information will show all of the plots you have on all of the remote harvesters.

Also see: #3875

It sounds like there is a patch coming soon that will relay this information.
